### PR TITLE
Added Horse, Race, and User models

### PR DESCRIPTION
**Pull request by hughthorn**

### CODE REVIEW

File: raceforcash/src/models/Horse.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+export interface Horse {
+    id: number;
+    name: string;
+    birthdate: Date;
+    breedId: number;
```
Comment (kRenelt) [inline raceforcash/src/models/Horse.ts]: server should send this as a string-breed

File: raceforcash/src/models/Horse.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    health: string;
```
Comment (kRenelt) [inline raceforcash/src/models/Horse.ts]: number

Comment (hughthorn) [reply raceforcash/src/models/Horse.ts]: the health should be a number you mean?

File: raceforcash/src/models/Horse.ts (hunk 1 above)
Comment (hughthorn) [reply raceforcash/src/models/Horse.ts]: Changed to string

File: raceforcash/src/models/Horse.ts (hunk 2 above)
Comment (hughthorn) [reply raceforcash/src/models/Horse.ts]: Changed health to number

File: raceforcash/src/models/Race.ts [hunk 3]
```diff
@@ -0,0 +1,6 @@
+export interface Race {
+    id: number;
+    name: string;
+    startTime: Date;
+    payouts: number;
```
Comment (kRenelt) [inline raceforcash/src/models/Race.ts]: This should be array

Comment (kRenelt) [reply raceforcash/src/models/Race.ts]: Add a horse array also

Comment (hughthorn) [reply raceforcash/src/models/Race.ts]: Made payouts an array, added a Payout interface within model.

**Review by kRenelt — APPROVED** [review]

**Pull request merged**